### PR TITLE
Resolve all warnings in Preferences (aka General) target

### PR DESCRIPTION
**Pull request by nevack**
Also renamed General target to Preferences and changed the way it's product is copied.

### CODE REVIEW

File: Preferences/Preferences/Base.lproj/Preferences.xib [hunk 1]
```diff
@@ -1,15 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <document type="com.apple.InterfaceBuilder3.Cocoa.XIB" version="3.0" toolsVersion="17701" targetRuntime="MacOSX.Cocoa" propertyAccessControl="none" useAutolayout="YES">
     <dependencies>
-        <deployment identifier="macosx"/>
```
Comment (nevack) [inline Preferences/Preferences/Base.lproj/Preferences.xib:4]: This line diff was produced by Xcode

File: Preferences/Preferences/OutputsArrayController.m [hunk 2]
```diff
@@ -1,5 +1,7 @@
 #import "OutputsArrayController.h"
 
+#import <CoreAudio/AudioHardware.h>
```
Comment (nevack) [inline Preferences/Preferences/OutputsArrayController.m:3]: For referencing function in `@selector`

File: Preferences/Preferences/Preferences_Prefix.pch [hunk 3]
```diff
@@ -3,5 +3,6 @@
 //
 
 #ifdef __OBJC__
+    #import <AssertMacros.h>
```
Comment (nevack) [inline Preferences/Preferences/Preferences_Prefix.pch:6]: Fixes deprecated logging functions usage warning

**Comment by nevack** [comment]
Most of the diff is {General->Preferences} rename

File: Preferences/Preferences/OutputsArrayController.m (hunk 2 above)
Comment (kode54) [reply Preferences/Preferences/OutputsArrayController.m:3]: Should be fine.

**Pull request merged**